sys/cddl/compat/opensolaris/sys/vnode.h

```diff
@@ -268,7 +268,9 @@ vn_rename(char *from, char *to, enum uio_seg seg)
 
 	ASSERT(seg == UIO_SYSSPACE);
 
-	return (kern_renameat(curthread, AT_FDCWD, from, AT_FDCWD, to, seg));
+	return (kern_renameat(curthread, AT_FDCWD, 
+			      cheri_ptr(from, strlen(from) + 1), AT_FDCWD, 
+			      cheri_ptr(to, strlen(to) + 1), seg));
```

Comment:
Continuation lines are 4-spaces
```suggestion
	return (kern_renameat(curthread, AT_FDCWD, 
	    cheri_ptr(from, strlen(from) + 1), AT_FDCWD, 
	    cheri_ptr(to, strlen(to) + 1), seg));
```

Reply:
In the `.clang-format` there is `ContinuationIndentWidth: 4`, but it seems not to fix this automatically. @arichardson  Does it work for you in clion? I was wondering if `.clang-format` doesn't cover all the cases, or if I have a faulty setup.

Reply:
My setup was faulty. Now it works!